dist/pinny.js

```diff
@@ -230,7 +235,13 @@
 
             // Block scrolling on anything but pinny content
```

Comment:
@Helen-Mobify With the new functionality, this comment is not quite accurate. Do you mind updating it to reflect your changes?